### PR TITLE
FAI-1849 FAI-1848 Changes related to Account deletion Journey

### PR DESCRIPTION
**Pull request by balaji-j**
https://skillsfundingagency.atlassian.net/browse/FAI-1848
https://skillsfundingagency.atlassian.net/browse/FAI-1849

### CODE REVIEW

File: src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiRequest.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using SFA.DAS.FAA.Domain.Interfaces;
+
+namespace SFA.DAS.FAA.Domain.Applications.GetSubmittedApplications
+{
+    public record GetSubmittedApplicationsApiRequest(Guid CandidateId) : IGetApiRequest
+    {
+        public string GetUrl => $"applications/submitted?candidateId={CandidateId}";
```
Comment (dashton82) [inline src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiRequest.cs:7]: I feel like the route for this should be more like:
```
users/account-deletion
```
so it matches whats happening in the UsersController

File: src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiResponse.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿using Newtonsoft.Json;
+
+namespace SFA.DAS.FAA.Domain.Applications.GetSubmittedApplications
```
Comment (dashton82) [inline src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiResponse.cs:3]: i think this needs to exist in the Users folder and relate more to something like
```
GetAccountDeletionApplicationsToWithdrawApiResponse
```

File: src/SFA.DAS.FAA.Domain/Configuration/FindAnApprenticeship.cs [hunk 3]
```diff
@@ -6,4 +6,5 @@ public class FindAnApprenticeship
     public string? RedisConnectionString { get; set; }
     public string? GoogleMapsApiKey { get; set; }
     public string? GoogleMapsId { get; set; }
+    public bool FeatureToggle { get; set; } = false;
```
Comment (dashton82) [inline src/SFA.DAS.FAA.Domain/Configuration/FindAnApprenticeship.cs:9]: maybe name to AccountDeletionFeature if we are going down the feature toggle route?

**Review by dashton82 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.FAA.Domain/Configuration/FindAnApprenticeship.cs (hunk 3 above)
Comment (balaji-j) [reply src/SFA.DAS.FAA.Domain/Configuration/FindAnApprenticeship.cs]: Yes, I Agree, Config name replaced to AccountDeletionFeature

File: src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiResponse.cs (hunk 2 above)
Comment (balaji-j) [reply src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiResponse.cs]: Moved and renamed as appropriate

File: src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiRequest.cs (hunk 1 above)
Comment (balaji-j) [reply src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiRequest.cs]: Route path changed in APIM & web

**Review by dashton82 — APPROVED** [review]

**Pull request merged**